third_party/terser/README.md

```diff
@@ -13,7 +13,7 @@ To make changes to this code please submit patches to
 https://github.com/emscripten-core/terser/ and then re-create this bundle
 using the following steps:
 
-  $ CI=1 npm run build
+  $ npx rollup -c
```

Comment:
Intentional change? What is the difference?

Reply:
(and I can then remove the comment below)